### ISSUE TITLE
Rotatium Bug

### ISSUE THREAD

**Issue by Supermafia00** [HippieStationArchive/HippieStation13-1#2714]
If you die while the rotatium effect is active the rotated screen stays there and does not go away unless you restart the game or the round ends.

**Comment by ArcLumin** [comment]
Makes sense, as the chem stops being processed when you die. I guess it could be fixed by checking if the holder is dead and reseting the rotation
